### ISSUE TITLE
Requesting a new release

### ISSUE THREAD

**Issue by Tri125** [dequis/purple-facebook#512]
Hello, is it possible to make a new release with the latest change?

I was hoping to benefit from the fixes concerning the thrift API which was fixed last year (https://github.com/dequis/purple-facebook/pull/480).

Thank you for your consideration.

**Comment by martinambrus** [comment]
I actually started to experience the same error fixed in the Thrift API myself now and it's very consistent - can't even log into FB anymore. Agree with the new release request.

**Comment by solbu** [comment]
I agree too. Most distros only update releases, and don't follow any of the commits.

In the mean time I have fixed the Mageia package (for mga7, mga8 and cauldron) by lifting and applying patches 18 and 19 from the patches folder.
Our Q&A will test it and approve them withing a few days, and then updated packages should reach the mirrors shortly after that.

**Comment by ae-bogdan** [comment]
Hey guys, if you were still looking for releases with the latest code:

https://github.com/ae-bogdan/libfacebook_pidgin

**Comment by e1000sav** [comment]
Thank you very much. It's working for me :)
